### ISSUE TITLE
will it restart itself when pc is restarted ?? 

### ISSUE THREAD

**Issue by yashgautam02** [UnamSanctam/SilentXMRMiner#513]

**Comment by UnamSanctam** [comment]
Firstly you should use the new miner https://github.com/UnamSanctam/SilentCryptoMiner, secondly if you enable "Startup" then it will start itself with Windows.

**Comment by yashgautam02** [comment]
> Firstly you should use the new miner https://github.com/UnamSanctam/SilentCryptoMiner, secondly if you enable "Startup" then it will start itself with Windows.

how to enable startup brother !?

**Comment by yashgautam02** [comment]
> Firstly you should use the new miner https://github.com/UnamSanctam/SilentCryptoMiner, secondly if you enable "Startup" then it will start itself with Windows.

and thank for fast reply

**Comment by UnamSanctam** [comment]
There's a "Tab" called "Startup" that you can enable it in, there's also some example settings at the top here: https://github.com/UnamSanctam/SilentCryptoMiner/wiki that you can load into the miner as an example (press "Load" on the "Main" tab).

**Comment by yashgautam02** [comment]
and wheres the tab sir 😅 
is it in the software !?

**Comment by UnamSanctam** [comment]
Yes, download the builder https://github.com/UnamSanctam/SilentCryptoMiner/releases

**Comment by yashgautam02** [comment]
bro i have quick heal antivirus in my pc and it is detecting this tool as a virus  
what to do now !?

**Comment by UnamSanctam** [comment]
Exclude the miner from it? It's a miner so it will be detected (like all miners).

**Comment by yashgautam02** [comment]
it wont harm my pc !? right 😅

**Comment by yashgautam02** [comment]
and bro one more thing  can i use it on nanopool !?

**Comment by UnamSanctam** [comment]
Haha no it won't harm your PC, it's open source so the code is publicly available. And yes you can use nanopool.
